src/xtgeo/grid3d/_find_gridprop_in_eclrun.py

```diff
@@ -395,10 +396,16 @@ def get_actnum_from_porv(init_filelike, grid):
     generator = filter_lgr(eclio.lazy_read(init_filelike))
     intehead, logihead, generator = peek_headers(generator)
 
+    if "INTERSECT" not in str(intehead.simulator):
+        # no need to continue if other simulators than INTERSECT
+        return
+
```

Comment:
Are we sure that this only applicable to Intersect case? The Eclipse manual specified that PORV can be used to identify active cells but didn't mention that it's Intersect specific. The implementation in ResInsight is also not restricted to Intersect.